prime-router/build.gradle.kts

```diff
@@ -0,0 +1,312 @@
+                        isFluentSetters = true
+                        isPojos = true
+                        isPojosEqualsAndHashCode = true
+                        isPojosToString = true
+                        isJavaTimeTypes = true
+                    }
+                    target.apply {
+                        packageName = jooqPackageName
+                        directory = jooqSourceDir
+                    }
+                }
+            }
+        }
+    }
+}
+
+// Set jOOQ task to participate in Gradle's incremental build feature
+tasks.named<nu.studer.gradle.jooq.JooqGenerate>("generateJooq") {
+    dependsOn("migrate")
+    allInputsDeclared.set(true)
+}
+
+// Convenience tasks
+tasks.register("compile") {
+    group = rootProject.description ?: ""
+    description = "Compile the code"
+    dependsOn("compileKotlin")
+}
+
+tasks.register("migrate") {
+    group = rootProject.description ?: ""
+    description = "Load the database with the latest schema"
+    dependsOn("flywayMigrate")
+}
+
+tasks.register("package") {
+    group = rootProject.description ?: ""
+    description = "Package the code and necessary files to run the Azure functions"
+    dependsOn("azureFunctionsPackage")
+    dependsOn("copyAzureResources")
+    dependsOn("copyAzureScripts")
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://kotlin.bintray.com/kotlinx")
+    }
+    maven {
+        url = uri("https://jitpack.io")
+    }
+}
+
+dependencies {
+    jooqGenerator("org.postgresql:postgresql:42.2.19")
+
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.32")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-common:1.4.32")
+    implementation("org.jetbrains.kotlin:kotlin-reflect:1.4.32")
+    implementation("com.microsoft.azure.functions:azure-functions-java-library:1.4.2")
+    implementation("com.azure:azure-core:1.15.0")
+    implementation("com.azure:azure-core-http-netty:1.9.1")
+    implementation("com.azure:azure-storage-blob:12.10.2") {
+        exclude(group = "com.azure", module = "azure-core")
+    }
+    implementation("com.azure:azure-storage-queue:12.8.0") {
+        exclude(group = "com.azure", module = "azure-core")
+    }
+    implementation("com.azure:azure-security-keyvault-secrets:4.2.7") {
+        exclude(group = "com.azure", module = "azure-core")
+        exclude(group = "com.azure", module = "azure-core-http-netty")
+    }
+    implementation("com.azure:azure-identity:1.2.5") {
+        exclude(group = "com.azure", module = "azure-core")
+        exclude(group = "com.azure", module = "azure-core-http-netty")
+    }
+    implementation("org.apache.logging.log4j:log4j-api:[2.13.2,)")
+    implementation("org.apache.logging.log4j:log4j-core:[2.13.2,)")
+    implementation("org.apache.logging.log4j:log4j-slf4j-impl:[2.13.2,)")
+    implementation("org.apache.logging.log4j:log4j-api-kotlin:1.0.0")
+    implementation("com.github.doyaaaaaken:kotlin-csv-jvm:0.15.1")
```

Comment:
Upgrade to 0.15.2.

Reply:
Updated.

Reply:
Will need to keep track of when https://github.com/CDCgov/prime-data-hub/pull/941 gets merged in.